.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,49 @@
+name: Maven e2e test - simple
```

Comment:
We don't usually set a name because the file name uniquely describes the test.
```suggestion
```

Reply:
Also it makes it easier to read the workflows from this page https://github.com/slsa-framework/example-package/tree/main/.github/workflows